blog/2025-01-15-service-info.md

```diff
@@ -0,0 +1,11 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "Relocate dhcp/ssdp/usb/zeroconf ServiceInfo models"
+---
+
+As of Home Assistant Core 2025.2, the following ServiceInfo models have been relocated:
```

Comment:
Please add a paragraph and an example what the developer needs to do to update their integration, and mention when the deprecation period ends.